modules/ai/module.php

```diff
@@ -148,6 +151,75 @@ public function __construct() {
 		} );
 	}
 
+	public function enqueue_ai_woocommerce() {
+
+		if ( ! class_exists( 'WooCommerce' ) || ! post_type_exists( 'product' ) ||
+			! isset( $_GET['post_type'] ) || 'product' !== $_GET['post_type'] ) {
+			return;
+		}
+
+		wp_enqueue_script( 'elementor-ai-unify-product-images',
+			$this->get_js_assets_url( 'ai-unify-product-images' ),
+			[
+				'jquery',
+				'elementor-v2-ui',
+				'elementor-v2-icons',
+			],
+			ELEMENTOR_VERSION,
+			true
+		);
+
+		wp_localize_script(
+			'elementor-ai-unify-product-images',
+			'UnifyProductImagesConfig',
+			[
+				'ajax_url' => admin_url( 'admin-ajax.php' ),
+				'nonce' => wp_create_nonce( 'elementor-ai-unify-product-images_nonce' ),
+			]
+		);
+
+		add_filter( 'bulk_actions-edit-product', function ( $data ) {
+			return $this->add_products_bulk_action( $data );
+		});
+
+		wp_set_script_translations( 'elementor-ai-unify-product-images', 'elementor' );
+
+	}
+
+	private function add_products_bulk_action( $bulk_actions ) {
+		$bulk_actions['elementor-ai-unify-product-images'] = __( 'Unify with Elementor AI', 'elementor' );
+		return $bulk_actions;
+	}
+
+	public function handle_unify_product_images_ajax() {
+		check_ajax_referer( 'elementor-ai-unify-product-images_nonce', 'nonce' );
+
+		$post_ids = isset( $_POST['post_ids'] ) ? array_map( 'intval', $_POST['post_ids'] ) : [];
+
+		if ( ! current_user_can( 'edit_products' ) ) {
+			wp_send_json_error( [ 'message' => 'You do not have permission to edit products' ] );
+		}
+
+		if ( empty( $post_ids ) ) {
+			wp_send_json_error( [ 'message' => 'No products selected' ] );
```

Comment:
Use translating function (try to find existing strings that are already translated)